### DISCUSSION TITLE
how can  I check  typst version and use conditionals in my template?

### DISCUSSION THREAD

**Discussion by danieltomasz** [typst/typst#5369] [Q&A] [answered]
I would like to work on typst template that people might use with quarto, quarto ships bundled typst version and people might have problem to update it;  how I can have conditional defintions  used on the basis of the quarto compiler template is using?
For example there is many deprecations and changes between 0.11.1 and 0.12 and  I would like to support both version of the typst in the same template;
````
 show par: set block(spacing: spacing) 
 ````

or 
````
 show par: set par(spacing: spacing) 
````
should be evaluated depending of the version of typst compiler

**Comment by MDLC01** [comment, marked as answer]
The `sys.version` constant contains the compiler version: https://typst.app/docs/reference/foundations/version/. But generally you shouldn't worry too much about older compiler versions. It is completely acceptable for packages to only support the latest compiler version.
